projects/elife.yaml

```diff
@@ -68,7 +68,9 @@ defaults:
             idle_timeout: 60
             certificate: arn:aws:iam::512686554592:server-certificate/cloudfront/wildcard.elifesciences.org/wildcard.elifesciences.org
             healthcheck:
+                protocol: http
                 port: 80
+                path: /ping
```

Comment:
`/ping` or `/status`? The former's only guaranteed to check that the service is running (though due to architecture checks a certain amount of components), but the latter should check (as much as possible) that every component is working (eg Journal checks that it can talk to the API).